### ISSUE TITLE
UI text scroll

### ISSUE THREAD

**Issue by ghost** [gap9269/SDDSound#149]

**Comment by gap9269** [comment]
I think this needs a softer sound

**Comment by ghost** [comment]
Definitely needs a change.  I dislike how you hear each individual click so clearly, so I'm going to make the next one have a longer tail so it all blends together nice and smooth

**Comment by gap9269** [comment]
This is still pretty clicky clacky. We might want to investigate a different option or axe it altogether.

**Comment by ghost** [comment]
I agree.  Weirdly the old one that wasnt meant for this worked better. I'll
try to make one more like that.

On Tue, Jul 21, 2015 at 6:36 PM, Gary notifications@github.com wrote:

> This is still pretty clicky clacky. We might want to investigate a
> different option or axe it altogether.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gap9269/SDDSound/issues/149#issuecomment-123526413.

## 

Miles Flanagan
Composer, Sound Designer for Games
milesflanaganmusic.com
